### PR TITLE
Update implementation of AMP analytics to reflect plugin changes

### PR DESCRIPTION
**Pull request by davisshaver**
See conversation at https://github.com/Automattic/amp-wp/issues/1496#issuecomment-428291725.

### CODE REVIEW

File: wp-parsely.php [hunk 1]
```diff
@@ -1392,14 +1379,18 @@ public function parsely_add_amp_actions() {
...
 			return $analytics;
 		}
 
 		$analytics['parsely'] = array(
-			'type'        => 'parsely',
-			'attributes'  => array(),
-			'config_data' => array(
+			'type'       => 'parsely',
+			'attributes' => array(),
+			'config'     => array(
```
Comment (davisshaver) [inline wp-parsely.php]: See here for discussion of how `config` may be preferred to `config_data` https://github.com/Automattic/amp-wp/issues/1498

File: wp-parsely.php [hunk 2]
```diff
@@ -124,7 +124,7 @@ public function __construct() {
 		add_action( 'wp_head', array( $this, 'insert_parsely_page' ) );
 		add_action( 'wp_footer', array( $this, 'insert_parsely_javascript' ) );
 		add_action( 'instant_articles_compat_registry_analytics', array( $this, 'insert_parsely_tracking_fbia' ) );
-		add_action( 'pre_amp_render_post', array( $this, 'parsely_add_amp_actions' ) );
```
Comment (davisshaver) [inline wp-parsely.php]: This action was "essentially deprecated"

**Comment by joshuarrrr** [comment]
Hey @davisshaver :wave: 

Thanks a lot for pulling this together! I'll check to see what it might take to get it into the next release.

cc @hbbtstar @jtotoole

**Comment by davisshaver** [comment]
Stand down on this for now, I think that some of these changes were too specific to the new native mode in that plugin.

**Pull request closed without merging**